Common/UnitDefinitions/Force.json

```diff
@@ -93,6 +103,11 @@
     {
       "SingularName": "Poundal",
```

Comment:
I see there is a notion of foot-pound-second system and centimeter-gram-second system (CGS / Gauss).
I'll create a PR to discuss adding those.

Reply:
Indeed there is. CGS, FPS, MKS